client/components/domains/connect-domain-step/transfer-domain-step-unlock.jsx

```diff
@@ -32,6 +32,10 @@ const TransferDomainStepUnlock = ( {
 	}, [ domain, setCheckInProgress ] );
 
 	const checkDomainLockStatus = async () => {
+		if ( initialDomainLockStatus === domainLockStatusType.UNKNOWN ) {
```

Comment:
```suggestion
		if ( domainLockStatusType.UNKNOWN === initialDomainLockStatus ) {
```
Minor, but I guess we should use "Yoda conditions" here.